src/components/signup/components/SignupForm.js

```diff
@@ -23,25 +23,23 @@ const {
 } = SIGNUP_STRINGS;
 
 function generateSignupInputs(decorator) {
-    return INPUTS.map(input => {
-        const { field, inputType, label, rules } = input;
-
-        return (
-            <FormItem
-                key={field}
-                {...FORM_ITEM_LAYOUT}
-                label={label}
-            >
-                {
-                    decorator(field, {
-                        rules,
-                    })(
-                        <Input type={inputType} />
-                    )
-                }
-            </FormItem>
-        );
-    });
+    return INPUTS.map(({ field, inputType, label, rules }) =>
```

Comment:
quick learner, nice use of destructuring here.

Reply:
Thank you 😊

Reply:
neat and concise

Reply:
well done